### ISSUE TITLE
ForAll as a child Element of Ol

### ISSUE THREAD

**Issue by ckosmowski** [speedata/publisher#444]
Long time no see, i am currently layouting around a bit and stumbled upon the following:

According to layoutschema-de.xsd "ForAll" is allowed as a child element of `Ul` and it is not allowed as a child element of `Ol`. However using ForAll as a child of `Ol` seems to work just fine.

So this works:

```xml
<Record element="bullet">
        <NewPage/>
        <PlaceObject area="text">
            <Textblock fontfamily="Baskerville">
                <Ol>
                    <ForAll select="entry">
                        <Li><Value select="."/></Li>
                    </ForAll>
                </Ol>
            </Textblock>
        </PlaceObject>
    </Record>
```

Is it missing in the schema only?

**Comment by pgundlach** [comment]
Hi CK! Nice to see you here!

This seems like a schema error. It will be fixed in the next version. I can send you the current version of the schema if you want.

**Comment by ckosmowski** [comment]
Nice to be here again :-)

I'll get it in the next version, i fixed it locally i just wasn't sure enough that is isn't intended to file a pull request. Next time i'll be happy to contribute.

**Comment by pgundlach** [comment]
Fixed with a94d3d175c4dc12b5550d4524a39aea207a9e63f
